Open_Source_Events/Cross WOC/README.md

```diff
@@ -0,0 +1,9 @@
+<h1 align="center"> Cross Winter of Code (CrossWoC) </h1>
```

Comment:
Change the name of the directory from `Cross WOC` to `Cross-Winter-of-Code` or `Cross-WOC`